CRM/Mailing/Event/BAO/Forward.php

```diff
@@ -100,8 +98,8 @@ public static function &forward($job_id, $queue_id, $hash, $forward_email, $from
     if (isset($dao->queue_id) ||
       (isset($dao->do_not_email) && $dao->do_not_email == 1)
     ) {
-      /* We already sent this mailing to $forward_email, or we should
-       * never email this contact.  Give up. */
```

Comment:
I didn't know this comment style was not in the style guides - learnt something!